src/services/rssServerService.ts

```
@@ -0,0 +1,182 @@
+  private app: express.Application;
+  private server: any;
+  private port: number;
+  private apiToken?: string;
+
+  capabilityDescription =
+    'HTTP server for serving RSS feeds and API endpoints.';
+
+  constructor(protected runtime: IAgentRuntime) {
+    super(runtime);
+    this.app = express();
+    this.port = parseInt(
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)